src/ExpressionEvaluator/Core/Source/ResultProvider/Helpers/DebuggerDisplayInfo.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.Debugger.Clr;
+using Microsoft.VisualStudio.Debugger.Evaluation.ClrCompilation;
+
+namespace Microsoft.CodeAnalysis.ExpressionEvaluator
+{
+    internal struct DebuggerDisplayInfo
+    {
+        public readonly DkmClrType TargetType;
+        public readonly DkmClrDebuggerDisplayAttribute Attribute;
+
+        public static readonly DebuggerDisplayInfo None = default(DebuggerDisplayInfo);
```

Comment:
I did it for readability, but in the current code, it's only used in 1 place, so it doesn't seem worth it...  I'll just inline it.